demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -541,7 +555,7 @@ export class DemoMeetingApp implements
             await this.openVideoInputFromSelection(camera, false);
             this.audioVideo.startLocalVideoTile();
           } catch (err) {
-            this.log('no video input device selected');
+            this.log(`failed to start local video tile due to ${err}`);
```

Comment:
is `` more costly ?

Reply:
It turns into a string concatenation, which isn't free, but cost isn't the main motivation here.

When a browser console logs an object (especially an error!), you get much more useful output: colorized, with error stacks, with clickable and explorable objects. You can assign the error to a variable, navigate to code, etc.

<img width="869" alt="Screen Shot 2020-12-02 at 9 29 24 AM" src="https://user-images.githubusercontent.com/40039049/100909000-04bf3f00-3481-11eb-98f3-1804dfdc13ce.png">


The SDK logging is pretty frustrating because it doesn't leverage this, but at least in the demo app we can do better.

Reply:
👍  make sense. thanks